CompetitionMars/CompetitionMars/Pages/CertificationPage.cs

```diff
@@ -0,0 +1,258 @@
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5); 
+
+            driver.Navigate().Refresh();
+
+
+            // }
+
+            // else 
+            // {
+            //  Console.WriteLine("Not found");
+            //}
+        }
+
+        public string GetUpdatedCertification()
+        {
+            IWebElement updatedCertificate = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td[1]"));
+            return updatedCertificate.Text;
+
+        }
+
+        public void DeleteCertification()
+        {
+           
+            certificationTab.Click();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+            JsonHelper jsonHelperObject = new JsonHelper();
+            string jsonFilePath = "C:\\internship notes\\CompetitionMars\\CompetitionMars\\CompetitionMars\\CompetitionMars\\TestData\\DeleteCertificateTestData.json";
+            
+            List<Certification> certificationTestData = jsonHelperObject.ReadCertificateTestDataFromJson(jsonFilePath);
+
+            Console.WriteLine(certificationTestData.ToString());
+
+            foreach (var certificate in certificationTestData)
+            {
+                string certificateName = certificate.Certificate;
+
+                //  Console.WriteLine(certificateName);
+                Wait.WaitToExist(driver, "XPath", "//th[text()='Certificate']//ancestor::thead//following-sibling::tbody[last()]/tr", 7);
+
+                IReadOnlyCollection<IWebElement> rows = driver.FindElements(By.XPath("//th[text()='Certificate']//ancestor::thead//following-sibling::tbody[last()]/tr"));
+                foreach (IWebElement row in rows) 
+                {
+                    driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(7);
+                    IWebElement certificateNameElement = row.FindElement(By.XPath("./td[1]"));
+                    string certificateToDelete = certificateNameElement.Text;
+
+                    if(certificateToDelete == certificate.Certificate) 
+                    {
+                        IWebElement deleteIcon = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody[last()]/tr/td[4]/span[2]/i"));
+                        deleteIcon.Click();
+                        driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+                      
+                      
+                    }
+                }
+               // driver.Navigate().Refresh();
+            }          
+
+        }
+
+        public string GetDeletedCertification()
+        {
+            JsonHelper jsonHelperObject = new JsonHelper();
+            string jsonFilePath = "C:\\internship notes\\CompetitionMars\\CompetitionMars\\CompetitionMars\\CompetitionMars\\TestData\\DeleteCertificateTestData.json";
```

Comment:
same here load test data from test or test steps, pages should only contain logic around elements